javascript/role.js

```diff
@@ -0,0 +1,509 @@
+roleDetails = JSON.parse(localStorage.getItem('roleDetails'));
+
+employee = JSON.parse(localStorage.getItem('employee'));
+createCards(roleDetails);
+
+document.addEventListener("click", (event) => {
```

Comment:
Create a function in common.js to handle this functionality. And call the function from its respective js file by passing some parameters.